src/mame/sega/model2.cpp

```diff
@@ -31,8 +31,6 @@
     - manxtt: no escape from "active motion slider" tutorial (needs analog inputs),
               bypass it by entering then exiting service mode;
     - manxtt: no bikes are visible (not a z-sort issue!);
-    - manxtt: course select island map doesn't rotate properly:
-              timing issue, i960 sends read/write geo addresses to bufferram 0x918000/4 which TGP never reads in time.
```

Comment:
The issue just above is also fixed since the TGP rewrite fwiw.

Reply:
Indeed it is. There's probably other issues that can be removed from the list; I need to do some testing